python/pyarrow/tests/test_dataset.py

```diff
@@ -2795,7 +2804,7 @@ def test_parquet_dataset_lazy_filtering(tempdir, open_logging_fs):
     # created with ParquetDatasetFactory from a _metadata file
 
     root_path = tempdir / "test_parquet_dataset_lazy_filtering"
-    metadata_path, _ = _create_parquet_dataset_simple(root_path)
+    metadata_path, _ = _create_parquet_dataset_simple(root_path, True)
```

Comment:
You can also add a default for this keyword in the helper function

Reply:
I added the default.